src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -22,6 +22,24 @@
             </Setter.Value>
         </Setter>
     </Style>
+    <Style TargetType="Border" x:Key="ContainedDialog">
+        <Setter Property="BorderBrush" Value="Black"/>
```

Comment:
Do the brushes in lines 26 and 30 need to respect any high contrast settings?

Reply:
Line 30, no, that's just for a fade effect. Line 26 is actually ignored because thickness is 0, so I will remove it entirely.

Reply:
Removed